Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+CREATE TABLE "MzTab-M"."Metadata_metabolomics" (
+    "mzTab-version" text NOT NULL,
+    title text,
+    description text,
+    sample_processing_number text,
+    sample_processing text,
+    "instrument_number-name" text,
+    "instrument-name" text,
+    "instrument_number-source" text,
+    "instrument-source" text,
+    instrument_number text,
+    analyzer_number text,
+    "instrument-analyzer" text,
+    "instrument_number-detector" text,
+    "instrument-detector" text,
+    software_number text NOT NULL,
+    software text NOT NULL,
+    setting_number text,
+    "software-setting" text,
+    publication_number text,
+    publication text,
+    "contact_number-name" text,
+    "contact-name" text,
+    "contact_number-affiliation" text,
+    "contact-affiliation" text,
+    "contact_number-email" text,
+    "contact-email" text,
+    uri_number text,
+    uri text,
+    external_study_uri_number text,
+    external_study_uri text,
+    quantification_method text NOT NULL,
+    sample_number text,
+    sample text,
+    species_number text,
+    "sample-species" text,
+    tissue_number text,
+    "sample-tissue" text,
+    cell_type_number text,
+    "sample-cell_type" text,
+    disease_number text,
+    "sample-disease" text,
+    "sample-description" text,
+    custom_number text,
+    "sample-custom" text,
+    ms_run_number text NOT NULL,
+    "ms_run-location" text NOT NULL,
+    "ms_run-instrument_ref" integer,
+    "ms_run-format" text,
+    "ms_run-id_format" text,
+    fragmentation_method_number text,
+    "ms_run-fragmentation_method" text,
+    scan_polarity_number text NOT NULL,
+    "ms_run-scan_polarity" text NOT NULL,
+    "ms_run-hash" text,
+    "ms_run-hash_method" text,
+    assay_number text NOT NULL,
+    assay text NOT NULL,
+    "assay-custom" text,
+    "assay-external_uri" text,
+    "assay-sample_ref" text,
+    "assay-ms_run_ref" text NOT NULL,
+    study_variable_number text NOT NULL,
+    study_variable text NOT NULL,
+    "study_variable-assay_refs" text NOT NULL,
+    "study_variable-average_function" text,
+    "study_variable-variation_function" text,
+    "study_variable-description" text NOT NULL,
+    "study_variable-factors" text,
+    "study-custom" text,
+    cv_number text NOT NULL,
+    "cv-label" text NOT NULL,
+    "cv-full_name" text NOT NULL,
+    "cv-version" text NOT NULL,
+    "cv-uri" text NOT NULL,
+    database_number text NOT NULL,
+    database text NOT NULL,
+    "database-prefix" text NOT NULL,
+    "database-version" text NOT NULL,
+    "database-uri" text NOT NULL,
+    derivatization_agent_number text,
+    derivatization_agent text,
+    "small_molecule-quantification_unit" text,
+    "small_molecule-identification_reliability" text,
+    id_confidence_measure_number text NOT NULL,
+    id_confidence_measure text NOT NULL,
+    "colunit-small_molecule" text,
+    "colunit-small_molecule_feature" text,
+    "colunit-small_molecule_evidence" text,
+    "ID" integer NOT NULL,
```

Comment:
Please check all of these tables for consistency when it comes to the internal ID.

Reply:
done